src/main/java/com/fundation/logic/view/resultTable/PopupMenu.java

```diff
@@ -62,5 +63,19 @@ public void mousePressed(MouseEvent e) {
         } catch (Exception e) {
             e.getMessage();
         }
+
+        try {
+            detailsItem.addMouseListener(new MouseAdapter() {
+                @Override
+                public void mousePressed(MouseEvent e) {
```

Comment:
Variable name not consistent with standars, please use a variable name more descriptive.

Reply:
done